Makefile

```diff
@@ -54,6 +54,11 @@ install: $(BUILD_SCRIPT)
 		-e "$$(perl -I lib -M"Munin::Common::Defaults" \
 			-e "Munin::Common::Defaults->print_as_sed_substitutions();")"
 
+.PHONY: perlcritic
```

Comment:
I would prefer to keep this in the `lint` target and maybe mask its error code until all issues are fixed?
Or maybe at least add a TODO for merging these targets, when the issues are fixed?

Reply:
added